platform/linuxbsd/os_linuxbsd.cpp

```diff
@@ -242,6 +246,200 @@ String OS_LinuxBSD::get_version() const {
 	return uts.version;
 }
 
+Vector<String> OS_LinuxBSD::get_video_adapter_driver_info() const {
```

Comment:
It won't fail. I use regex as an optional verification. Similar to how it's used for email validation. Not perfect, but better than nothing.  
If regex moduls is disabled, it will simply not verify the return value.